torch/ao/quantization/backend_config/executorch.py

```diff
@@ -56,6 +56,13 @@
     is_dynamic=True,
 )
 
+executorch_weight_only_quint8_dtype_config = DTypeConfig(
```

Comment:
would be great to have a TODO to rename this file, since there is no executorch backend.  Maybe it should be `executorch_with_qnnpack_backing`?

when other targets will start to be supported by executorch runtime (DSP chips, etc), they may have different capabilities compared to what is being added here